addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -507,6 +507,9 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
             top: '+5px',
         });
         this.$el.append($button);
+        var options = $.summernote.eventHandler.modules.linkDialog.options || {};
```

Comment:
What about this? This is just a proposition 🙂 

```suggestion
        $.summernote.eventHandler.modules.linkDialog.options = Object.assign({},
            $.summernote.eventHandler.modules.linkDialog.options,
            {forceNewWindow: true});
```

Reply:
Done ! :) Thanks for the tip ! :)